src/testController.ts

```diff
@@ -201,45 +201,58 @@ export class TestController {
     await this.telemetry.sendCodeLensEvent("test");
     const run = this.testController.createTestRun(request, undefined, true);
     const queue: vscode.TestItem[] = [];
+    const enqueue = (test: vscode.TestItem) => {
+      queue.push(test);
+      run.enqueued(test);
+    };
 
     if (request.include) {
-      request.include.forEach((test) => queue.push(test));
+      request.include.forEach(enqueue);
     } else {
-      this.testController.items.forEach((test) => queue.push(test));
+      this.testController.items.forEach(enqueue);
     }
 
     while (queue.length > 0 && !token.isCancellationRequested) {
       const test = queue.pop()!;
 
       if (request.exclude?.includes(test)) {
+        run.skipped(test);
         continue;
       }
+      run.started(test);
 
       if (test.tags.find((tag) => tag.id === "example")) {
         const start = Date.now();
         try {
-          await this.assertTestPasses(test);
+          const output: string = await this.assertTestPasses(test);
+          run.appendOutput(output, undefined, test);
```

Comment:
Hmm this is what I see
<img width="1089" alt="image" src="https://github.com/Shopify/vscode-ruby-lsp/assets/43872616/9a273fe5-da9b-46f2-82a0-85b22ccee422">

Reply:
I guess we can trim the lines returned at https://github.com/Shopify/vscode-ruby-lsp/blob/4ae202d7b748dade7ab3f5f0ef4b2c3c9da7361f/src/testController.ts#L264-L268

Now it just returns whatever the stdout is.

Does this happen when you run the test in terminal?

Reply:
No, only on the test results tab.

Reply:
It still happens. Interestingly, if I click on `+1 more line` it then displays the text properly and I can no longer get into the weird state.

This must be something with my setup. Let's revert this last commit since it didn't really change it and merge.